### DISCUSSION TITLE
O Roadmap para novos contribuidores

### DISCUSSION THREAD

**Discussion by elvitin** [lucashpmelo/node-mde#10] [Ideas]
Se for interessante, elaborar um Roadmap contendo os principais conhecimentos exigidos para contribuição, exemplo, quais principais documentos por onde começar, etc. Mas tudo alinhado com a filosofia e o futuro pretendido da biblioteca.

Acho que seria interessante para excelentes desenvolvedores JavaScript, porém com o primeiro contado nesse ecossistema fiscal Brasileiro.

**Comment by lucashpmelo** [comment]
Ainda tenho a ideia de escrever uma Wiki explicando como a Consulta e o Manifesto de notas funciona, e listando quais as Notas Técnicas são necessárias a leitura para um melhor entendimento do funcionamento dos Web Services.

A biblioteca já está completa a algum tempo, e nos últimos meses só foi necessário realizar atualizações de dependências para evitar vulnerabilidades, então talvez seja a próxima coisa que irei fazer.
